pkg/install/v1alpha1/cstor_pool_0.7.0.go

```diff
@@ -461,7 +469,7 @@ spec:
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
 metadata:
-  name: cstor-pool-delete-deletestoragepoolcr-default-0.7.0
+  name: cstor-pool-delete-deletestoragepoolcr-default
 spec:
   meta: |
     id: deletestoragepoolcr
```

Comment:
I believe it would be a good idea to separate CAS templates from codes. This will be helpful for releasing the patches for CAS templates very easily to users for older versions too in future. An approach for this could be to host the CAS templates in some remote repo from where it can be pulled by the API server once deployed.

Reply:
> @AmitKumarDas we are declaring the templates as the constants, so it will be just sharing the memory and good for compiler optimizations but it is not known how much memory it will consume. Can we do benchmarks for use of constants vs files, what do you say ?
> Just thinking from the memory overhead perspective, as applying yamls is one time process and will not be used frequently. Correct me if i'm wrong.

1/ Need to revisit the approach based on findings & other aspects. Will raise a issue.

**NOTE:** Following aspects came to my mind leading to embed the yaml in go code were:
- developers need not worry about writing a .yaml file that might necessitate use of tools / unit test to validate the same yamls
- need not change the Dockerfile to include these artifacts (though it will be one time only!! Perhaps go has a way to find these files from the package folder which means no need to bother about Dockerfile)
- no need to deal with file io package etc.

Yamls can keep growing & hence issues of memory consumption (& eventual garbage collection) based on:
1. multiple versions of openebs
2. increasing number of openebs components
3. increasing number of runtasks/castemplates